website/static/js/fangorn.js

```diff
@@ -1478,7 +1478,7 @@ var FGItemButtons = {
                     }, 'Delete'));
 
             }
-        } else if(item.data.provider) {
+        } else if(item.data.provider && item.children.length != 0) {
```

Comment:
Use `!==` for comparison. 

Please use jshint to catch errors like this in the future.